src/tokenContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import getToken from './lib/token';
+
+const TokenContext = React.createContext();
```

Comment:
I think we could get by with using this as a simple hook rather than Context.

This [gist](https://gist.github.com/stevelikesmusic/5ac2b8df4e3c1e204aa031def6335dde) should get you the same results without creating a new Context. It's a hook that still keeps everything in state. 

The only difference is that we'd read from localStorage a little more depending on how often the hook is used. Not a bad thing here per se, but one tradeoff to be mindful of if you were scaling out this app.